algo/src/main/java/org/neo4j/graphalgo/AllShortestPathsProc.java

```diff
@@ -47,6 +51,8 @@
                     .load(configuration.getGraphImpl());
 
             return new MSBFSAllShortestPaths(graph, Pools.DEFAULT)
+                    .withLog(log)
+                    .withKernelTransaction(transaction)
```

Comment:
it would be nice of the API of the Algos would be the TerminationFlag interface and we would do the wrapping in the proc.